library/js/oeUI/universalTooltip.js

```diff
@@ -8,24 +8,5 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 $(function () {
-    $(document).tooltip({
-        show: {
-            delay: 400
-        },
-        hide: {
-            delay: 0
-        },
-        position: {
-            my: "center top",
-            at: "center bottom-5",
-            collision: "flipfit"
-        }
-    });
-    $(this).click(function () {
-        $(this).tooltip({
-            hide: {
-                delay: 0
-            }
-        });
-    });
```

Comment:
This should've been completely deleted with the references to the file also deleted if you decided to remove this file.